docs/getting-started/build-your-own-development-automation.md

```diff
@@ -217,6 +225,175 @@ Click on `Close` (you could also close the issue in GitHub if you prefer) and se
+</div>
+
+Merge the new PR into your project:
+
+<div class="ss-container">
+  <img src="../images/merge-convert-pull-request.png" alt="Merge the editor's PR into your project" class="ss-small">
+</div>
+
+Now update your local copy of the project and you should see that the `.atomist` directory has been added. Atomist keeps all of its files in the `.atomist` directory so that there should be no conflicts with your project's files.
+
+ Execute `rug test` in your project's directory to ensure that everything is working for your new Rug generator project:
+
+```
+> rug test
+Resolving dependencies for yourorg:dotnetgettingstarted:0.1.0 ← local completed
+Invoking TypeScript Compiler on ts script sources
+  Created .atomist/editors/NewDotNetASPCoreProject.js
+  Created .atomist/editors/NewDotNetASPCoreProject.js.map
+Processing script sources completed
+Loading yourorg:dotnetgettingstarted:0.1.0 ← local into runtime completed
+Executing scenario NewDotNetASPCoreProject should create a new project based on this archive...
+  Testing assertion fileExists(SimpleLiteral(.gitignore))
+  Testing assertion fileExists(SimpleLiteral(Program.cs))
+  Testing assertion fileExists(SimpleLiteral(README.md))
+  Testing assertion fileExists(SimpleLiteral(Startup.cs))
+  Testing assertion fileExists(SimpleLiteral(project.json))
+Running test scenarios in yourorg:dotnetgettingstarted:0.1.0 ← local completed
+
+Successfully executed 1 of 1 scenarios: Test SUCCESS
+```
+
+To see what's been added, use the Rug CLI by executing `rug shell -l` and then `describe archive` to see the contents of your new Rug archive:
+
+```
+rug → describe archive
+Restoring yourorg:dotnetgettingstarted:0.1.0 ← local into runtime completed
+
+yourorg:dotnetgettingstarted:0.1.0
+
+→ Origin
+  git@github.com:yourorg/dotnetgettingstarted.git#master (79e34ae)
+→ Archive
+  ~/code/src/github.com/yourorg/dotnetgettingstarted/ (761 kb in 126 files)
+
+→ Generators
+  NewDotNetASPCoreProject
+    Sample TypeScript generator used by AddNewDotNetASPCoreProject
+
+→ Requires
+  [0.12.0,1.0.0)
+
+To get more information on any of the Rugs listed above, run:
+  describe editor|generator|executor|reviewer ARTIFACT
+yourorg:dotnetgettingstarted:0.1.0 ← local
+```
+
+Congratulations, you now have a new generator. It's usually a good idea at this point to give things a test run and so run the generator in the `rug shell` using the `generate` command, for example:
+
+```
+rug → generate NewDotNetASPCoreProject newdotnetservice -C ../
+```
+
+The `-C` changes the directory to above where you're running the shell for the output of the generator. Go now and check out the output and make sure it is still a working project. You should now be able to change directory to where you generated your new project and be able to test and run it as usual.
+
+That's the new generator created, now it's time to publish it so you can list and execute it using `@atomist` from your team's slack.
```

Comment:
We've been using "the Atomist Bot" instead of "`@atomist`" because the bot could have a different name.